mod.ts

```diff
@@ -0,0 +1 @@
+export { CorsMiddleware } from "./src/cors/cors_middleware"
```

Comment:
this file isn't needed. users will have to `import { CorsMiddleware } from "https://deno.land/x/drash-middleware@v{version}/cors/mod.ts"`;